bundles/core/src/main/java/com/adobe/cq/forms/core/components/models/form/Field.java

```diff
@@ -93,4 +95,14 @@ default Object[] getDefault() {
         return null;
     }
 
+    @JsonView(Views.Author.class)
+    default boolean getDorExclusion() {
```

Comment:
This needs to be part of fd:dor namespace as custom properties. You can check existing implementation of custom properties related to layout, https://github.com/adobe/aem-core-forms-components/blob/dev/bundles/core/src/main/java/com/adobe/cq/forms/core/components/models/form/Base.java#L340-L349

In a similar manner we could add getDorProperties in base class with JsonView set to Author.

Reply:
This would have jsonIgnore annotation, since it would be part of `fd:dor` namespace. Each piece of code in core components needs to have proper java doc.

Reply:
Sure, I will incorporate the comments.